packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -52,10 +52,16 @@ const startDateTimeTimestamp = new Date().setMinutes(
   new Date().getMinutes() + 30
 );
 
+const BalanceDivider = {
+  max: 1,
+  half: 2,
+  quarter: 4,
```

Comment:
Can we make it uppercase to show that's a constant? Was thinking about using an enum but this works fine too.

Reply:
I can.

Reply:
fixed!